packages/file-upload/src/FileUpload.tsx

```diff
@@ -313,13 +313,7 @@ export const fileInputStyles = ({
 					backgroundColor: boxPalette.systemSuccessMuted,
 					borderColor: boxPalette.systemSuccess,
 			  }
-			: disabled
-			? {
-					cursor: 'not-allowed',
-					opacity: 0.3,
-					background: 'none',
-			  }
 			: undefined),
 
-		'&:focus': packs.outline,
```

Comment:
This element could never receive focus, so it doesn't make sense to have focus styles